omod/src/main/resources/config.xml

```diff
@@ -42,7 +42,7 @@
     <conditionalResources>
     	<conditionalResource>
             <path>/lib/metadatasharing-api-1.9-${project.parent.version}.jar</path>
-            <openmrsVersion>1.9.9 - 1.12.*</openmrsVersion>
+            <openmrsVersion>1.9.9 - 1.12*</openmrsVersion>
```

Comment:
This should be `1.9.9 - 1.12.*`. It won't work properly without the dot.

Reply:
Sorry let me do it